f8a_report/main.py

```diff
@@ -22,18 +22,24 @@ def main():
 
     start_date = (today - timedelta(days=1)).strftime('%Y-%m-%d')
     end_date = today.strftime('%Y-%m-%d')
-    response, ingestion_results = r.get_report(start_date, end_date, 'daily')
+    response, ingestion_results = r.get_report(start_date, end_date, 'daily', retrain='F')
```

Comment:
It's always better to use boolean values `retrain=False`. And everywhere else.

Reply:
ok sure, will do it